rts/builds/dedicated/CMakeLists.txt

```diff
@@ -62,12 +62,7 @@ include_directories(${ENGINE_SRC_ROOT_DIR}/lib/asio/include)
 
 
 ### Assemble sources
-if	(APPLE)
-	set(sources_engine_Platform_CrashHandler
-		${ENGINE_SRC_ROOT_DIR}/System/Platform/CpuID.cpp
-		${ENGINE_SRC_ROOT_DIR}/System/Platform/Threading.cpp
-		${ENGINE_SRC_ROOT_DIR}/System/Platform/Mac/CrashHandler.cpp)
```

Comment:
What about the files in `/System/Platform/Mac/`?

What about other random `#if apple` checks in source, such as
https://github.com/beyond-all-reason/spring/blob/5c84984cd5f4afd800b993b02dec05d16a8ff199/rts/Game/UI/HwMouseCursor.cpp#L43

Reply:
Good point. These also can be removed. My intention was to make PR small and limited in scope. Here I focused on cmake. I thought that way it's easier to review. Should I add wider scope here?

Reply:
Yep, let's keep it focused on CMakeLists for now